### PR TITLE
Fixed the problem that the parameters torch.max and paddle.max were inconsistent

### PR DESCRIPTION
**Pull request by wfs2010**
Fixed the problem that the parameters torch.max and paddle.max were inconsistent.
# for the parameters input and dim, #modify to the corresponding parameters x and dim. 
——————————————————————————————
# pytorch code
import torch
import torch.nn as nn
import torch.optim as optim
import torch.nn.functional as F

image = torch.randn(4, 4)
torch.max(image, dim=1)

torch.max(input=image, dim=1)
torch.min(image)

torch.min(image, dim=1)
——————————————————————————————
# paddle code
import paddle
image = paddle.randn(shape=[4, 4])
paddle.max(image, axis=1)
paddle.max(x=image, axis=1)
paddle.min(x=image)
paddle.min(image, axis=1)

### CODE REVIEW

File: paconvert/api_matcher.py [hunk 1]
```diff
@@ -332,8 +332,21 @@ def get_paddle_nodes(self, args, kwargs):
         # return (values, indices) and paddle not implement, not convert
         if len(args) > 1 and isinstance(args[1], ast.Num):  
             return None
+
         if 'dim' in new_kwargs:
-            return None
+            for k, v in enumerate(kwargs):
```
Comment (zhouwei25) [inline paconvert/api_matcher.py:337]: torch是返回两个Tensor，我们返回一个，这个对得上？
